db/raw/add-hotel-plugin.sql

```diff
@@ -0,0 +1,1576 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+--
+-- Name: sqitch; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA sqitch;
```

Comment:
Ok, that's good to know.  I probably could have figured that out for myself by reading the ``sqitch`` docs more carefully.  I skimmed through them and it wasn't obvious whether they were sequential migrations (like ``alembic``) or dependency-based migrations (like Django), or completely unordered migrations (like a lot of one-off migrations scripts).

Thanks for the clarification.